source/components/organisms/Step/Step.js

```diff
@@ -132,9 +133,35 @@ function Step({
         formNavigation.back();
       };
 
+  const [returnScrollY, setReturnScrollY] = useState(0);
+  const scrollRef = React.useRef();
+
+  const handleFocus = (e, isSelect = false) => {
+    const scrollResponder = scrollRef.current.getScrollResponder();
+
+    e.target.measure((x, y, width, height, pageX, pageY) => {
+      let keyboardHeight = 0;
+
+      if (scrollResponder.keyboardWillOpenTo) {
+        keyboardHeight = scrollResponder.keyboardWillOpenTo.startCoordinates.height;
+      }
+
+      setReturnScrollY(pageY + height);
+
+      const scrollToY = pageY + height + keyboardHeight;
+
+      if (isSelect) scrollResponder.props.scrollToPosition(0, scrollToY);
+    });
+  };
+
   return (
     <StepContainer>
-      <KeyboardAwareScrollView keyboardShouldPersistTaps="always">
+      <KeyboardAwareScrollView
+        keyboardShouldPersistTaps="always"
+        resetScrollToCoords={(() => ({ x: 0, y: returnScrollY }))()}
```

Comment:
What is the reason for wrapping the value of `resetScrollToCoords` in an anonymous function and calling it? Doesn't `resetScrollToCoords={{ x: 0, y: returnScrollY }}` work?

Reply:
Sorry, old habit. Your proposed method should work the same.